### PR TITLE
PR to collect glob strings

### PR DESCRIPTION
**Pull request by DatGizmo**
This is the PR setup commit to collect glob strings

Signed-off-by: Sebastian Wezel <sebastian.wezel@nordicsemi.no>

### CODE REVIEW

**Comment by DatGizmo** [comment]
@Qung @ardo-nordic @jorgenmk @eibe85  @elisabeth-solheim @andvib @rafpelc

Hello everybody.
This PR is used to collect all the missing glob strings to automatically add labels for testing.
You can push your changes to this branch. I'll then take care of finalising the PR.

Pleas have the changes available by tomorrow (20.04) EOD

**Comment by DatGizmo** [comment]
@ChristopherBittner @magnev @wiminordic @rakons @maciekrozanski @jangalda-nsc 
Forgot to add you in the comment above.

**Review by thst-nordic — APPROVED** [review]

**Review by mswarowsky — APPROVED** [review]

File: .github/labeler.yml [hunk 1]
```diff
@@ -44,3 +53,14 @@
 "CI-tfm-test":
   - "modules/tfm/**/*"
   - "samples/tfm/**/*"
+  - "tests/subsys/spm/secure_services/**/*"
```
Comment (oyvindronningstad) [inline .github/labeler.yml:56]: SPM secure services are unrelated to TFM
```suggestion
```

**Review by oyvindronningstad — CHANGES_REQUESTED** [review]

File: .github/labeler.yml (hunk 1 above)
Comment (andvib) [reply .github/labeler.yml:56]: It's not related to TF-M, but this test is still executed on the TF-M CI as there weren't really any other meaningful place to execute it. So even though it looks out of place I think we should keep it here, at least until we have a clearer scope for were the SPM tests will be executed.

Comment (oyvindronningstad) [reply .github/labeler.yml:56]: Aah ok, then of course we keep it.

**Review by oyvindronningstad — APPROVED** [review]

**Review by ChristopherBittner — APPROVED** [review]

**Review by andvib — APPROVED** [review]

**Pull request merged**